torch/distributed/distributed_c10d.py

```diff
@@ -3726,6 +3752,8 @@ def barrier(group=GroupMember.WORLD, async_op=False, device_ids=None):
 
 def monitored_barrier(group=GroupMember.WORLD, timeout=None, wait_all_ranks=False):
     """
+    Synchronize processes.
```

Comment:
Can this be replaced with the first sentence from below?

```
Synchronizes all processes similar to ``torch.distributed.barrier``, but takes
    a configurable timeout and is able to report ranks that did not pass this
    barrier within that timeout.

Reply:
Thant will be very long sentence and so it will inconsistent with doc format. Considering your suggestion, I am thinking to change it this way: 

```
Synchronize processes similar to ``torch.distributed.barrier``, but consider a configurable timeout.

It is able to report ranks that did not pass this barrier within the provided timeout.
Specifically,.........
```
Hope that make sense but if not I will update. Thanks @H-Huang